tensorrt_llm/_torch/auto_deploy/transformations/_graph.py

```diff
@@ -89,28 +89,47 @@ def lift_to_meta(model: nn.Module, strict_missing: bool = True, strict_unexpecte
         )
 
 
+def named_graphmodules(gm: fx.GraphModule) -> Iterator[Tuple[str, fx.GraphModule]]:
+    """Yield (name, submodule) for every fx.GraphModule inside gm (including gm itself)."""
+    for name, m in gm.named_modules():
+        if isinstance(m, fx.GraphModule):
+            yield name, m
+
+
 def move_to_device(gm: fx.GraphModule, device: DeviceLikeType) -> fx.GraphModule:
```

Comment:
Can you do a separate private function `_move_single_gm_to_device` without recursion? This would look useful to me